### ISSUE TITLE
Division via /

### ISSUE THREAD

**Issue by steve-py96** [blueprintui/blueprintui#85]
using the current sass version with a self-compiled blueprint.scss spams the following warning into the console:

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(..., ...)
```

their [auto-migrator](https://sass-lang.com/documentation/breaking-changes/slash-div#automatic-migration) is pretty neat and running it once over here would prolly be a nice quickwin!

**EDIT**: forgot to add the current workaround... :p
as in other similar issues also mentioned the best current workaround is to downgrade sass to v1.32.x (via `yarn add -D sass@1.32.0` f.e.) since the warning came with v1.33.0 (version findable on the migrator link also)

**Comment by coryrylan** [comment]
Thanks for the heads up. Hoping to get a patch release soon

**Comment by nlappe** [comment]
Can confirm this.
The deprecation warning kinda hit me in the face after upgrading to Angular 13 (which also pulls sass version 1.44 with it)

**A quick and dirty solution that i tested in my node_modules folder.
Regex search & replace over all files.
search: (\$\w+)\/(\$\w+)
replace with: calc(#{$1}\/#{$2})**

this made the warnings disappear and everything looks the same as before but idk if it may break some tests or something.
(i tried to clone the repo and create a PR, but i cant get the thing to work my system. Seems like my local node version doesnt work with the repo and i cant get the enviroment straight in docker. Tests wont work, building errors out etc... so the regex S&R is all i can offer right now)

Also i would to express my gratitude. blueprint is the easiest and smallest grid framework i've found so far. 
Thank you for creating it.

**Comment by coryrylan** [comment]
Thanks for the feedback! Hoping to release a patch over holiday break. I ran into this as well with Angular 13.

**Comment by coryrylan** [comment]
Just released 3.1.2 which should fix this issue now.
